tests/e2e/test_rosacli_ingress.go

```diff
@@ -284,4 +285,83 @@ var _ = Describe("Edit default ingress",
 					Expect(expectLabel).To(BeElementOf(ingressRouteSelectors))
 				}
 			})
+		It("can update ingress components (oauth, downloads, console) - [id:72868]",
+			labels.Medium,
+			labels.Runtime.Day2,
+			func() {
+
+				By("Record ingress default value")
+				output, err := rosaClient.Ingress.ListIngress(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+				ingressList, err := rosaClient.Ingress.ReflectIngressList(output)
+				Expect(err).ToNot(HaveOccurred())
+				defaultIngress := ingressList.Ingresses[0]
+
+				By("Check edit ingress help message")
+				output, err = rosaClient.Ingress.EditIngress(clusterID, "-h")
+				Expect(err).ToNot(HaveOccurred())
+				Expect(output.String()).Should(ContainSubstring("--component-routes"))
+
+				By("Edit ingress with --component-routes")
+				componentRoutes := "oauth: hostname=oauth.hostname.com;tlsSecretRef=oauth-secret,downloads: hostname=downloads.hostname.com;tlsSecretRef=downloads-secret,console: hostname=console.hostname.com;tlsSecretRef=console-secret"
+				output, err = rosaClient.Ingress.EditIngress(clusterID,
+					defaultIngress.ID,
+					"--component-routes", componentRoutes,
+				)
+				Expect(err).ToNot(HaveOccurred())
+				defer rosaClient.Ingress.EditIngress(clusterID,
+					defaultIngress.ID,
+					"--component-routes", "oauth: hostname=oauth.hostname.com;tlsSecretRef=oauth-secret,downloads: hostname=downloads.hostname.com;tlsSecretRef=downloads-secret,console: hostname=console.hostname.com;tlsSecretRef=console-secret",
+				)
+
+				By("List ingress to check")
+				output, err = rosaClient.Ingress.ListIngress(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+			})
+		It("cannot update ingress components with incorrect syntax - [id:72868]",
```

Comment:
Combine the `It` specs to only one. Our rule need to follow one case one It spec